bin/templates/ext/yarp/api_node.c.erb

```diff
@@ -90,8 +92,9 @@ yp_node_new(yp_parser_t *parser, yp_node_t *node, rb_encoding *encoding, ID *con
     }
 }
 
+#line <%= __LINE__ + 1 %> "<%= File.basename(__FILE__) %>"
 VALUE yp_ast_new(yp_parser_t *parser, yp_node_t *node, rb_encoding *encoding) {
-    ID constants[parser->constant_pool.size];
+    ID * constants = calloc(parser->constant_pool.size, sizeof(ID));
```

Comment:
Super surprised this can't be used, I thought this was a C99 feature?

Reply:
I think on windows machines we can't dynamically allocate the size  of the array